ocaml/idl/datamodel.ml

```diff
@@ -2274,6 +2283,18 @@ module Tunnel = struct
         ; field ~ty:tunnel_protocol ~default_value:(Some (VEnum "gre"))
             ~lifecycle:[(Published, "1.250.0", "Add protocol field to tunnel")]
             "protocol" "The protocol used for tunneling (either GRE or VxLAN)"
+        ; field ~qualifier:StaticRO ~ty:Int ~default_value:(Some (VInt 0L))
+            ~lifecycle:[(Published, rel_next, "Set vxlan id to tunnel")]
```

Comment:
The  documentation is confusing. This is a read-only value but the documentation uses active language: "set ... to". The documentation needs to describe what the observed int value means: "if true, <this and that happens or is also true>".

Reply:
I'd like to fix it.